### PR TITLE
Remove hardcoded right-click action in AnnoCanvas

### LINKED ISSUE #266: Right click action to remove hovered object is still hardcoded
The logic to remove an object via right-click is still hardcoded - this should be moved to a hotkey, so it can be rebound. It also makes it a little clear if the user wishes to bind a different action to right-click.

image

### PR DESCRIPTION
**Pull request by AgmasGold**
Fixes #266.

### CODE REVIEW

File: AnnoDesigner/AnnoCanvas.xaml.cs [hunk 1]
```diff
@@ -2080,6 +2079,19 @@ private void ExecuteDuplicate(object param)
             }
         }
 
+        private readonly Hotkey deleteHoveredObjectHotkey;
+        private readonly ICommand deleteHoveredObjectCommand;
+        private void ExecuteDeleteHoveredObject(object param)
+        {
+            var obj = GetObjectAt(_mousePosition);
+            if (obj == null)
```
Comment (FroggieFrog) [inline AnnoDesigner/AnnoCanvas.xaml.cs:2087]: I think this should be `if (obj != null)`.

File: AnnoDesigner/Localization/Localization.cs [hunk 2]
```diff
@@ -343,6 +344,7 @@ public static void Init(ICommons commons)
                         { "Delete" , "Löschen" },
                         { "Duplicate" , "Duplikat" },
                         { "RotateAll" , "Alle drehen" },
+                        { "DeleteHovered" , "Schwebeobjekt löschen" },
```
Comment (FroggieFrog) [inline AnnoDesigner/Localization/Localization.cs:347]: Maybe "Objekt unter Mauszeiger löschen"? (english would be something like "Delete object under mouse cursor")
The name "Schwebeobjekt" is not really used/known in german.

Comment (AgmasGold) [reply AnnoDesigner/Localization/Localization.cs]: I think I prefer "Delete object under cursor" anyway - I'll update the english translation as well.

**Comment by AgmasGold** [comment]
Just realised a minor change - this is now effectively firing on mouse down, rather than mouse up. Its not a huge issue, and users can just rebind the command if they want.

**Review by FroggieFrog — APPROVED** [review]
Good job 🎉

**Pull request merged**